Lib/urllib/parse.py

```diff
@@ -364,11 +364,19 @@ def _fix_result_transcoding():
 del _fix_result_transcoding
 
 def urlparse(url, scheme='', allow_fragments=True):
-    """Parse a URL into 6 components:
+    """Parse URL and return a ParseResult.
```

Comment:
I find this first line less clear than the original (though it is technically correct.)

How about:
```python
"""Parse a URL into 6 components:
<scheme>://<netloc>/<path>;<params>?<query>#<fragment>

The result is a ParseResult object, which is a named 6-tuple with
fields corresponding to the above.

...
```

This then followed by the later description of the other input parameters and the final note regarding % escapes not being expanded.

(The same goes for `urlsplit()` as well.)